surveys/2021-annual-survey/questions.md

```diff
@@ -1,19 +1,138 @@
 # Survey questions
 
-## About you
+### Do you use Rust?
 
-See [who](./design/who.md).
+Select one:
 
-The following are primarily for cohort analysis, secondarily for understanding the shape of the community.
+- Yes, I have used for Rust in the past three months
+- I have used Rust in the past, but not in the past three months
+- No, I have never used Rust
 
-Are you a Rust user
+> Rephrased answers slightly to be more precise.
+> TODO is three months the right amount of time
+> 
+> **justification**
+>
+> fundamental for cohort analysis
 
-- I have used for Rust for any reason in the past three months
-- I have used Rust in the past, but not in the past three months
-- I have never used Rust
+## For previous Rust users
+
+### How long did you use Rust before you stopped?
+
+Select one:
+
+> **REMOVED**
+>
+> Not useful
+
+### How long ago did you stop using Rust:
+
+Select one:
+
+> **REMOVED**
+>
+> Not useful
+
+### Why did you stop using Rust?
+
+Select all that apply:
+
+- Rust didn't solve a problem for me
```

Comment:
I really don't like this answer as I'm not sure what it actually means...

Reply:
In general I'd like to think through whether the group of answers we have here is the right one.

Reply:
Yeah, I was wondering about that. I'm not a great fan of the current set, but I figure they were good enough and it would be good to have continuity for this question